ros/lib/utils.py

```diff
@@ -31,6 +31,21 @@ def get_or_create(session, model, keys, **kwargs):
     return instance
 
 
+def delete_record(session, model, keys, **kwargs):
```

Comment:
will it be good to rename `kwargs` to something specific? OR add details to method docstring?

Reply:
renamed to `system_fields`.

Reply:
As per Suraj's suggestion [here](https://github.com/RedHatInsights/ros-backend/pull/55/#discussion_r610355151), I'll keep it to kwargs but, add details to the method.